src/common-utils/main.sh

```diff
@@ -455,20 +457,27 @@ if [ "${INSTALL_ZSH}" = "true" ]; then
             -c fetch.fsck.zeroPaddedFilemode=ignore \
             -c receive.fsck.zeroPaddedFilemode=ignore \
             "https://github.com/ohmyzsh/ohmyzsh" "${oh_my_install_dir}" 2>&1
-        echo -e "$(cat "${template_path}")\nDISABLE_AUTO_UPDATE=true\nDISABLE_UPDATE_PROMPT=true" > ${user_rc_file}
-        sed -i -e 's/ZSH_THEME=.*/ZSH_THEME="devcontainers"/g' ${user_rc_file}
 
         # Add Dev Containers theme
         mkdir -p ${oh_my_install_dir}/custom/themes
         cp -f "${FEATURE_DIR}/scripts/devcontainers.zsh-theme" "${oh_my_install_dir}/custom/themes/devcontainers.zsh-theme"
         ln -s "${oh_my_install_dir}/custom/themes/devcontainers.zsh-theme" "${oh_my_install_dir}/custom/themes/codespaces.zsh-theme"
 
+        # Attempt adding devcontainers .zshrc, does nothing given an existing file
+        if [[ (! -f "$user_rc_file") && ("$INSTALL_OH_MY_ZSH_CONFIG" = "true") ]]; then
```

Comment:
> (! -f "$user_rc_file") 

Question: Provided `installOhMyZshConfig:true`, shouldn't the contents be simply overwrriten by this Feature? If a user does not want that behavior, they can always pass `installOhMyZshConfig:false`